libs/react-core/src/lib/provider/notistack-provider.tsx

```diff
@@ -0,0 +1,124 @@
+import { ReactNode, useRef } from 'react';
+import { SnackbarProvider, SnackbarKey } from 'notistack';
+import { useTheme } from '@mui/material/styles';
+import { GlobalStyles, Collapse, Box, IconButton, alpha } from '@mui/material';
+
+import {
+  Info as InfoIcon,
+  CheckCircle as CheckCircleIcon,
+  Warning as WarningIcon,
+  Error as ErrorIcon,
+} from '@mui/icons-material';
+import { Icon } from '@admin/app/components/icon';
+
+
+function SnackbarStyles() {
+  const theme = useTheme();
+
+  const isLight = theme.palette.mode === 'light';
+
+  return (
+    <GlobalStyles
+      styles={{
+        // '#root': {
+        '& .notistack-Snackbar': {
+          width: '100%',
+          maxWidth: 500,
+          '& .notistack-MuiContent':
+          {
+            color: theme.palette.text.primary + '!important',
+            backgroundColor: theme.palette.background.paper + '!important',
+            borderRadius: '8px !important',
+          },
+          [theme.breakpoints.up('md')]: {
+            minWidth: 240,
+          },
+        },
+        '& .SnackbarItem-message': {
+          padding: '0 !important',
+          fontWeight: theme.typography.fontWeightMedium,
+        },
+        // '& .SnackbarItem-action': {
+        //   marginRight: 0,
+        //   color: theme.palette.action.active,
+        //   '& svg': { width: 20, height: 20 },
+        // },
+        // },
+      }}
+    />
+  );
+}
+
+
+type Props = {
```

Comment:
Give proper name i.e ` NotistackProviderProps` and use interface instead of type. Also make export

Reply:
fix name